### DISCUSSION TITLE
How `Fund Features` works?

### DISCUSSION THREAD

**Discussion by ratson** [admob-plus/admob-plus#314] [Fund Features]
Category `Fund Features` is used to keep track for features that the community want to be implemented.

In order to make that happen, @ratson is asking if the community would back the feature with at least $1,000 dollars for his work.

Leave comment on each feature discussion with amount you would like to contribute, when promised amount is over $1000, @ratson will ask for the funding and start the implementation.

Here is the feature list,
- [Mediation Support #269](https://github.com/admob-plus/admob-plus/discussions/269) ![](https://progress-bar.dev/35)
- [Native Ads #311](https://github.com/admob-plus/admob-plus/discussions/311) ![](https://progress-bar.dev/100)
- [Support for app open ad unit #313](https://github.com/admob-plus/admob-plus/discussions/313) ![](https://progress-bar.dev/100) `admob-plus-cordova@1.13.0`
- [Google Ad Manager #354 ](https://github.com/admob-plus/admob-plus/discussions/354) ![](https://progress-bar.dev/0)

_the bar represents the promised donation progress_

What do you think about this idea?

**Comment by EMI-INDO** [comment]
@ratson 

For any of the features that will be implemented I'm willing to give $50-$100
To reach $1000 requires ten or more participants.

Is this possible?

**Reply by Carlosps** [reply]
I am also willing to pay $50-$100 depending on the feature

**Reply by sksk008** [reply]
currently i am contribute $10 but if any new update/features I am also willing to pay $50.

**Reply by eliadAfeka** [reply]
@ratson I am also willing to pay $50-$100 depending on the feature

**Reply by creakosta** [reply]
@ratson ready pay for adding features $100-$150

**Reply by eliadAfeka** [reply]
@ratson is this progress bar represent the developing progress or contribute progress?
if it's contribution progress, can you please open a place where we can donate and see how far are you from the goal, maybe we will contribute more.

**Reply by eliadAfeka** [reply]
@ratson please let me know how much missing in all of them so I will complete the the rest for you to get 1000$

**Reply by ratson** [reply]
@eliadAfeka You are so generous. I need to implement the features one by one anyway, why not you pick the first one for me to work on and I ask for the funding in the thread?

The missing amount could be calculated via $1000 * (1 - progress / 100), e.g. Mediation work $1000 * (1 - 35/100) = $650

**Reply by eliadAfeka** [reply]
@ratson I'm using this plugin for few years, so the minimum that I can do is to contribute when ever you ask. 
after you will be done with both features, i will become a friendly sponsor too.  
anyway, the open ad is high priority for me, and i think it will the easy one for you, so it will be great if you can start with it.
highly appreciate your work! :-)

**Reply by aymank900** [reply]
@ratson i'm willing to pay for Mediation Support 100$

**Reply by ghost** [reply]
@ratson  Support 100$
